CompetitionMARS/Tests/EducationTests.cs

```diff
@@ -0,0 +1,334 @@
+﻿using CompetitionMARS.Utilities;
+using CompetitionMARS.Pages;
+using CompetitionMARS.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.ComponentModel.DataAnnotations;
+using AventStack.ExtentReports.Reporter;
+using AventStack.ExtentReports;
+using static System.Net.Mime.MediaTypeNames;
+using System.Reflection;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+using RazorEngine;
+using OpenQA.Selenium.Chrome;
+using AventStack.ExtentReports.Model;
+
+
+namespace CompetitionMARS.Tests
+{
+    [TestFixture]
+
+    public class EducationTests : CommonDriver
+    {
+        private Education eduPage;
+        public static ExtentReports extent;
+
+        public List<string> addedEducationEntries = new List<string>();
+
+        [OneTimeSetUp]
```

Comment:
your hooks should be moved into a hooks class, for example OneTimeSetUp and OneTimeTearDown should be implemented in hooks your other hooks like setup and teardown can be per test class can you please change this